src/actions/guides/frontend/rendering_html.cr

```diff
@@ -399,21 +400,18 @@ class Guides::Frontend::RenderingHtml < GuideAction
 
     ### Rendering page specific content in the layout
 
-    Let's start with a quick example. In newly generated Lucky projects your
-    layouts call `shared_layout_head`. This method is defined in
-    `Shared::LayoutHead`. If you look in the `src/components/shared/layout_head`
-    file you'll see that it defines and calls a `page_title` method.
-
-    This means that you can customize the page title in your page by writing a
-    `page_title` method:
+    Let's start with a quick example. In newly generated Lucky projects
+    you'll see that `MainLayout` defines a `page_title` method that is passed to
+    the `Shared::LayoutHeader` component. Any page that inherits from `MainLayout`
+    can write a new `page_title` method to override the one in `MainLayout`.
```

Comment:
We use a component now. 

Also we should just focus on the `page_title` method since that's what this section is about. Using methods to render stuff in layouts